AppCenter/AppCenter/Internals/Model/CommonSchema/MSProtocolExtension.m

```diff
@@ -31,7 +34,9 @@ - (BOOL)isEqual:(id)object {
     return NO;
   }
   MSProtocolExtension *protocolExt = (MSProtocolExtension *)object;
-  return ((!self.devMake && !protocolExt.devMake) ||
+  return ((!self.ticketKeys && !protocolExt.ticketKeys) ||
+          [self.ticketKeys isEqualToArray:protocolExt.ticketKeys]) &&
```

Comment:
tabbing is weird for this return statement, did you run the clang formatter?

Reply:
This is correct as per our new formatting.